### ISSUE TITLE
Pocketcasts Support

### ISSUE THREAD

**Issue by nswint** [tgwizard/sls#444]
Request for support of scrobbling via Pocketcasts.

**Comment by nswint** [comment]
I was able to get now playing scrobbles using Pocketcast but not the
scrobble after it was completed.

The 2019 MTV VMAs by Pop Culture Happy Hour, Pop Culture Happy Hour

**Comment by goodevilgenius** [comment]
With v1.5.8-pre-v8, I upgraded, and then cleared my data.

When I opened the app back up, my scrobble cache was gone, and I had to log in again.

And now, scrobbles are finally happening successfully.

**Comment by a93h** [comment]
Yes. There is an impossible to fix sqlite bug with foreign keys.
I cannot migrate the database of users to the new setup.
It will wipe any recent scrobbles sitting in the cache unfortunately, but the app should not crash for most users.

**Comment by a93h** [comment]
You should not need to clear your data anymore.

**Comment by nswint** [comment]
> You should not need to clear your data anymore.
> This upgrade will have it happen automatically.

pre-v8 scrobbles automatically and forcibly now.   I did have to clear the data.  pre-v1 worked for me.  Then scrobbling broke from pre-v8 to pre-v7.  I skipped pre-v7.1.

**Comment by a93h** [comment]
pre-v8 Scrobble is not working?

**Comment by nswint** [comment]
No, they submit to now playing but do not scrobble at the specified
scrobble point. They just queue up in the scrobble cache.  I'm pretty sure
that I was impacted by whatever sqlite issue that was going on.  I had 48
scrobbles in the cache that I was waiting to submit but I bit the bullet
and deleted the data and reauthenticated.

**Comment by a93h** [comment]
Was music playing when you installed?

**Comment by nswint** [comment]
There was likely music or a podcast paused and resident in the notification
tray.
